f2/src/forms/WhatWasAffectedForm.js

```diff
@@ -145,10 +145,6 @@ export const WhatWasAffectedForm = props => {
                 </Alert>
               </Control>
             ) : null}
-
-            <P>
```

Comment:
please remove the import {p} on the top since it is not used.

Reply:
thanks! change made!